src/Functional/HStringFilter.php

```diff
@@ -6,35 +6,35 @@
 class HStringFilter
 {
     /** @var HString */
-    protected $string;
+    protected $hSTring;
```

Comment:
Small typo: `hSTring` should probably be `hString`

Renaming protected class properties is considered a BC break. In this instance any class extending HStringFilter and using `$this->string` would require updating their code to use `$this->hSTring` as the new property name. (But I don't think this class is likely to be extended in any other user's code so it doesn't need documenting this time :D )